### PR TITLE
Fix homepage title

### LINKED ISSUE #424: Homepage has bad title
"Accelerated Mobile Pages Project – Accelerated Mobile Pages Project"

Lets fix the redundancy and maybe add AMP (if we always append the string, lets use AMP there).

### PR DESCRIPTION
**Pull request by bpaduch**
Fixes #424 

to: @cramforce

### CODE REVIEW

File: views/partials/head.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <meta name="description" content="{% if doc.description %}{{doc.description}} – {% endif %}Accelerated Mobile Pages Project">
```
Comment (cramforce) [inline views/partials/head.html]: Also fix this.

File: content/pages/home.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-$title: Accelerated Mobile Pages Project
```
Comment (cramforce) [inline content/pages/home.html]: Lets keep this for now. Works well with the `- AMP` that is added automatically

File: views/partials/head.html (hunk 1 above)
Comment (cramforce) [inline views/partials/head.html]: Also wrap the whole tag in `if doc.description`. Much better to have none, than a generic one.

File: content/pages/home.html (hunk 2 above)
Comment (bpaduch) [inline content/pages/home.html]: Done

File: views/partials/head.html (hunk 1 above)
Comment (bpaduch) [inline views/partials/head.html]: Done and tested locally.

Comment (bpaduch) [inline views/partials/head.html]: Done

File: content/pages/home.html (hunk 2 above)
Comment (bpaduch) [inline content/pages/home.html]: Done

**Pull request merged**